mono/profiler/Makefile.am

```diff
@@ -94,5 +95,8 @@ else
 check-local: testlog
 endif
 
+suppressiondir = $(datadir)/mono-$(API_VER)/mono/profiler
+suppression_DATA = mono-profiler-logging.suppression
```

Comment:
should probably be `mono-profiler-log.suppression`, as `mono-profiler-logging.c` is old dead code